internal/internal.go

```diff
@@ -22,14 +22,20 @@ func ConfigDir() (string, error) {
 		return configDir, nil
 	}
 
-	if runtime.GOOS == "windows" {
-		return filepath.Join(os.Getenv("APPDATA"), "gops"), nil
+	if cd := os.Getenv("GOPS_CONFIG_DIR"); cd != "" {
```

Comment:
Could we please do some early returns here e.g
```go
if configDir := os.Getenv("GOPS_CONFIG_DIR"); configDir != "" {
      return configDir, nil
}
```
and the test of the code continues as it was before.

Reply:
eh, i need to set the `ConfigDir` option when use the `agent` , because set env `GOPS_CONFIG_DIR` in prod instance is not allowed.

Reply:
@zuoRambo thanks for your patience and for the edits.
Oh, okay thought the problem was that your $HOME directory couldn't be retrieved.

Alright in that case with this new information, here are some suggestions:
1) Sure, let's allow setting ConfigDir but modify https://github.com/google/gops/pull/61/files#diff-6ea70e0a4de93ebcb83000a028edc163R76 to something like this
```go
gopsDir := opts.ConfigDir
if configDir == "" {
   cfgDir, err := internal.ConfigDir()
   if err != nil {
       return err
   }
   gopsDir = cfgDir
}
```

2. Let's remove https://github.com/google/gops/pull/61/files#diff-7f51cfde52dc24f331c0c04f62ce53fbR49

3. We can keep https://github.com/google/gops/pull/61/files#diff-336a19a3907c67c768036d0fae6aa384R35

4. Could we make https://github.com/google/gops/pull/61/files#diff-6ea70e0a4de93ebcb83000a028edc163R47
```
// ConfigDir is the directory to store the configuration file,
// PID of the gops process, filename, port as well as content.
```